packages/env/lib/env.js

```diff
@@ -33,6 +33,8 @@ module.exports = {
 	 * @param {Object} options.spinner A CLI spinner which indicates progress.
 	 */
 	async start( { spinner } ) {
+		module.exports.stop( { spinner } );
```

Comment:
Why is this necessary? An inline comment might be helpful for future readers.

We'll also need to `await` this as `stop()` is an async function.